src/definitions.ts

```diff
@@ -669,6 +676,16 @@ export interface ICombinedTokenContext extends ITokenContext {
 }
 
 /**
+ * Facebook native login field permissions
+ */
+export type FacebookScopes = 'basic' | 'public_profile' | 'email';
```

Comment:
Drop the s, this is a union type! It represents a singular value, and the name should be singular